dev/CommonStyles/MediaTransportControls_themeresources.xaml

```diff
@@ -371,29 +371,38 @@
                                                 </AppBarButton.Icon>
                                             </AppBarButton>
                                             <AppBarSeparator x:Name="LeftSeparator" Height="0" Width="0" Margin="0,0" />
-                                            <AppBarButton x:Name="StopButton" Icon="Stop" Style="{StaticResource AppBarButtonStyle}" MediaTransportControlsHelper.DropoutOrder="7" Visibility="Collapsed" />
+                                            <AppBarButton x:Name="StopButton" Icon="Stop" Style="{StaticResource AppBarButtonStyle}" MediaTransportControlsHelper.DropoutOrder="7" Visibility="Collapsed">
+                                            </AppBarButton>
```

Comment:
I think you can revert this one -- although should it have a FontIcon just like all the others?

Reply:
Reverted. So turns out the solid stop button isn't part of the Segoe Fluent fonts, so using the stop symbol icon here is fine.